backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -602,6 +602,7 @@ public async System.Threading.Tasks.Task<Project> SetStatusAsync(Project project
                 .Include(s => s.Tasks)
                 .FirstOrDefault(s => s.Id == project.StatusId) ?? throw new KeyNotFoundException();
 
+            IEnumerable<int> incompleteTaskIds = new List<int>();
```

Comment:
Why pull this out of the `if` block?  I don't think we test for incomplete tasks if the status doesn't change.

Reply:
Additionally if a variable is only for private use there isn't much value in declaring an `IEnumerable<>` and then setting it as a `List<>`.   It just makes it more difficult to use.  Might as well go with `var` then you have all the access to `List` functionality natively.

Reply:
Right, I originally pulled this out because I was using the list in the DIS calculations below but was able to refactor to be more generic. Will remove.